{"name": "Невозможно обновить данные без авторизации", "status": "passed", "parameters": [{"name": "field", "value": "'name'"}], "start": 1742756529528, "stop": 1742756530594, "uuid": "2c796a00-b5dd-4c9c-bcaf-3b589f5197ea", "historyId": "2419109320a1266cce3693df83c2114e", "testCaseId": "50ab14e5bd59ba94665da529c21fdea1", "fullName": "tests.test_user.TestUser#test_update_unauthorized_user_data_fails", "labels": [{"name": "feature", "value": "Пользователь"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "LAPTOP-QVE1U26H"}, {"name": "thread", "value": "3900-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}